applications/luci-app-vpnbypass/luasrc/model/cbi/vpnbypass.lua

```diff
@@ -75,9 +72,7 @@ d = Map("dhcp")
 s4 = d:section(TypedSection, "dnsmasq")
 s4.anonymous = true
 di = s4:option(DynamicList, "ipset", translate("Domains to Bypass"),
-		translate("Domains to be accessed directly (outside of the VPN tunnel), see ")
-		.. [[<a href="]] .. readmeURL .. [[#bypass-domains-formatsyntax" target="_blank">]]
-		.. translate("README") .. [[</a> ]] .. translate("for syntax"))
+		translatef("Domains to be accessed directly (outside of the VPN tunnel), see %sREADME%s for syntax", "<a href=\"" .. readmeURL   .. "#bypass-domains-formatsyntax" .. "\" target=\"_blank\">", "</a>"))
```

Comment:
You might avoid really long lines, just for style.

Reply:
Please check https://github.com/openwrt/luci/pull/4001/files#diff-fab29750b070d6cc59baa0c9738196dcR75-R76 -- I'm not sure if I should split up the localizable content to make the line shorter than 100 characters.

Reply:
No need.
The next i18n-sync run will change the .pot .po lines in any case if they are too long.

I meant just that moving the translatef arguments to the next line, which you have now done. ;-)
(the first commit had that line with 200 chars...)